test_scripts/API/VehicleData/StabilityControlsStatus/012_SubscribeVehicleData_StabilityControlsStatus_and_other_parameters.lua

```diff
@@ -0,0 +1,68 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check subscription on StabilityControlsStatus and other parameters data
+--   via SubscribeVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+
+-- Steps:
+-- 1) App sends SubscribeVehicleData request with (stabilityControlsStatus = true, speed = true) to SDL
+-- SDL does:
+--  - send VehicleInfo.SubscribeVehicleData request with (stabilityControlsStatus = true, speed = true) to HMI
+-- HMI sends VehicleInfo.SubscribeVehicleData response "SUCCESS" with
+--   (stabilityControlsStatus = { resultCode = "SUCCESS", dataType = "VEHICLEDATA_STABILITYCONTROLSSTATUS"},
+--   speed = { resultCode = "SUCCESS", dataType = "VEHICLEDATA_SPEED"})
+-- SDL does:
+--  - send SubscribeVehicleData response with (success: true, resultCode: "SUCCESS") and received from HMI data to App
+-- 2) HMI sends VehicleInfo.OnVehicleData notifications with StabilityControlsStatus and Speed data
+-- SDL does:
+--  - send OnVehicleData notification with received from HMI data to App
+-- 3) HMI sends VehicleInfo.OnVehicleData notification with GPS data
+-- SDL does:
+--  - not send OnVehicleData notification with received from HMI data to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Local variables ]]
+local vdName1 = "gps"
+local vdName2 = "stabilityControlsStatus"
+local mobileRequest = {
+    [vdName1] = true,
+    [vdName2] = true
+  }
+local hmiRequest = mobileRequest
+local hmiResponse = {
+    [vdName1] = common.getResponse("SUCCESS", vdName1),
+    [vdName2] = common.getResponse("SUCCESS", vdName2)
+  }
+local mobileResponse = { 
+    success = true,
+    resultCode = "SUCCESS",
+    [vdName1] = common.getResponse("SUCCESS", vdName1),
+    [vdName2] = common.getResponse("SUCCESS", vdName2)
+  }
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Start SDL, init HMI, connect default mobile", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("PTU", common.policyTableUpdate, { common.ptUpdate })
+common.Step("Activate App", common.activateApp)
+
+common.Title("Test")
+common.Step("Subscribe on VehicleData",
```

Comment:
```suggestion
common.Step("Subscribe on VehicleData " .. vdName1 .. " " .. vdName2,
```
I would propose to add vdNames

Reply:
@VjKlepikov please check https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/commits/e86562fbe628dad0e5104d3c6551760077385116